src/Simplify.cpp

```diff
@@ -3908,6 +3924,68 @@ class Simplify : public IRMutator {
             IRMutator::visit(op);
         }
     }
+    template <typename T>
+    Expr hoist_slice_vector(Expr e) {
```

Comment:
I use T only for being able to use its "make" member. Is there any other cleaner way of doing this?